src/main/java/com/google/devtools/build/lib/analysis/platform/ConstraintCollection.java

```diff
@@ -43,10 +43,17 @@
   @Nullable private final ConstraintCollection parent;
   private final ImmutableMap<ConstraintSettingInfo, ConstraintValueInfo> constraints;
 
+  /**
+   * Create a new {@link ConstraintCollection} with the given constraint values.
```

Comment:
Nit: Method descriptions (including constructors) should start with an informative ("Creates") rather than an imperative ("Create") verb.

Reply:
Done.